tests/jobs/test_job_decorator.py

```diff
@@ -1184,28 +1184,29 @@ async def job_task(self) -> None:
             self.coresys.jobs.current.stage = "work"
 
     test = TestClass(coresys)
-
-    with time_machine.travel(dt):
-        job, _ = await test.job_scheduler()
```

Comment:
Technically, putting the rest of the existing function inside that `with` would fix the test already.

This change makes the test also a bit faster.